### DISCUSSION TITLE
Version 7.44.0-rc.1

### DISCUSSION THREAD

**Discussion by bluebill1049** [react-hook-form/react-hook-form#10275] [General]
📄 Form Component (#9735)
- add try and catch for JSON.stringify

🐞 fix #10139 with errors diff from the previous with field array action (#10216)
🐞 related #10238 return default values in watch and `useWatch` when reset is called with an empty object

thanks to @devakrishna33

<hr /><em>This discussion was created from the release <a href='https://github.com/react-hook-form/react-hook-form/releases/tag/v7.44.0-rc.1'>Version 7.44.0-rc.1</a>.</em>

**Comment by kasparkallas** [comment]
Shouldn't `useFormContext` also work with `TTransformedValues`?

**Reply by bluebill1049** [reply]
Yea, we can add that as well. Want to send us a PR?
